hypervisor/lxc/lxc.go

```diff
@@ -56,6 +60,9 @@ var settings struct {
 func init() {
 	hypervisor.RegisterProvider("lxc", &LXCHypervisorProvider{})
 	viper.SetDefault("hypervisor.script-path", "/etc/openvdc/scripts")
+	viper.SetDefault("hypervisor.image-server", "127.0.0.1/images")
```

Comment:
this item should be the URI syntax.

``hypervisor.image-server`` -> ``hypervisor.image-server-uri``